session/dialog_handler.go

```diff
@@ -0,0 +1,28 @@
+package session
+
+import (
+	"github.com/mysterium/node/communication"
+)
+
+// NewDialogHandler constructs handler which get all incoming dialogs does starts handling them
```

Comment:
// NewDialogHandler constructs handler which gets all incoming dialogs and starts handling them

Reply:
Fixed